src/client/Components/Util/LogTable.jsx

```diff
@@ -50,12 +50,16 @@ export default class LogTable extends React.Component {
 
     console.log(this.state.componentLogs);
 
-    const logTextLines = this.state.componentLogs.map((entry) => {
+    const tableEntries = this.state.componentLogs.map((entry) => {
       if (entry.log_text != null) {
-        return entry.log_text.split('\n');
+        return {
+          execution_num: entry.execution_num,
+          log_lines: entry.log_text.split('\n'),
+          log_error: entry.log_error
+        };
       }
-      return [];
-    });
+      return { execution_num: entry.execution_num, log_lines: [], log_error: entry.log_error };
+    }).filter((entry) => entry.execution_num !== 0);
```

Comment:
Little bit of repetition with the object creation here. Not sure if there is a clean fix, so not blocking PR

Reply:
Reasonable comment, but given that I'm overhauling this page not sure it's worth resolving at the moment. Will keep this in mind during the front-end work.